Header header
float64[] x
float64[] y
float64[] angle
float64[] area
float64[] ecc